client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -258,11 +217,11 @@ export class GoogleAnalyticsForm extends Component {
 								</FormSettingExplanation>
 							</fieldset>
 						) }
-						{ showAnalyticsForStores && (
+						{ siteIsJetpack && wooCommerceActive && (
 							<FormAnalyticsStores
 								fields={ fields }
 								handleToggleChange={ this.handleToggleChange }
-								showEnhanced={ showEnhancedAnalyticsForStores }
+								showEnhanced={ siteIsJetpack }
```

Comment:
nit: `showEnhanced` is effectively always `true`.

Reply:
👍 Cleaned up in https://github.com/Automattic/wp-calypso/pull/30042/commits/cffa1cb93f4a290c044c0f2672b5fd468704e8f2